ocaml/idl/datamodel_errors.ml

```diff
@@ -1203,8 +1203,8 @@ let _ =
     ~doc:"The current version of Xen or its control libraries is incompatible with the Toolstack." ();
 
   (* repository and updates errors *)
-  error Api_errors.set_repository_in_progress []
-    ~doc:"The operation could not be performed because a repository setup is in progress." ();
+  error Api_errors.configure_repositories_in_progress []
+    ~doc:"The operation could not be performed because repositories setup is in progress." ();
```

Comment:
The following sounds a bit more natural:

```suggestion
  error Api_errors.repository_configuration_in_progress []
    ~doc:"The operation could not be performed because another repository is already being configured" ();
```